src/content/migrate/4.md

```diff
@@ -11,23 +11,30 @@ This guide only shows major changes that affect end users. For more details plea
 
 ## Node.js v4
 
-If you are still using Node.js v4 or lower, you need to upgrade your Node.js installation to Node.js v6 or higher.
+*TLDR:; Node.js versions lower than 6 are no longer supported. So, upgrade to the latest Node.js LTS verison.*
 
+If you are still using Node.js v4 or lower, you need to upgrade your Node.js installation to Node.js v6 or higher. Ideally you should be getting the latest Long Term Support(LTS) version of Node.js. Instructions for upgrading your Node.js version can be found [here](https://stackoverflow.com/questions/10075990/upgrading-node-js-to-latest-version). This is because the source of webpack was changed to a higher ECMAScript version in webpack 4.
 
 ## CLI
 
 The CLI has moved to a separate package: webpack-cli. You need to install it before using webpack, see [basic setup](/guides/getting-started/#basic-setup).
 
+The installation guide can be found [here](/src/content/guides/installation.md).
```

Comment:
Did you try to follow this link? Is it working locally?

Reply:
Corrected.